src/test/java/com/iexec/common/docker/client/DockerClientInstanceTests.java

```diff
@@ -411,6 +413,21 @@ public void shouldPullImage() {
         dockerClientInstance.removeImage(ALPINE_LATEST);
     }
 
+    @Test
+    public void shouldPullImageWithExplicitTimeout() {
+        assertThat(dockerClientInstance.pullImage(ALPINE_LATEST, Duration.of(1, ChronoUnit.MINUTES))).isTrue();
```

Comment:
We may want to use a longer timeout to avoid unwanted failures due to slow network or loaded host.

Reply:
As this is a UT, I don't think we'd like to use a longer timeout. Or we should not always launch this test.

Reply:
Sorry, I will never call a test pulling an image from dockerhub a unit test.
Those tests are integration tests, maybe at the wrong place, but they are integration tests.
If we want unit tests, they need to be entirely rewritten and it is a great undertaking :-)